Release/snowplow mobile/0.1.0

emielver opened the pull request.

colmsnowplow reviewed (approved):
LGTM! Two things to point out/discuss on Monday:

1. There's no automated tests like this one: https://github.com/snowplow/dbt-snowplow-web/blob/main/.github/workflows/pr_tests.yml 

There may be practical limitations here but let's discuss, at the very least I'd be more comfortable with a firm plan of action on this.

2. We need to make sure that any change to the semver in the utils package is amended in this PR before release. It's obvious but calling it out explicitly because oversights can happen. :) 

Looking forward to covering both PRs and the release process on Monday

emielver commented:
Thanks for the review @colmsnowplow! I've updated the `packages.yml` to include the latest version of the `snowplow-utils` package that was just released, and I've added an [issue](https://github.com/snowplow/dbt-snowplow-mobile/issues/5) to implemented automated testing in the repo. If all is good let me know and I will release the package!

colmsnowplow commented:
LGTM @emielver, fire ahead! Thanks for the great work
